src/sentry/conf/server.py

```diff
@@ -69,6 +69,10 @@ def env(key, default="", type=None):
 IS_DEV = ENVIRONMENT == "development"
 
 DEBUG = IS_DEV
+if IS_DEV:
+    # Enable line buffering for stderr, TODO(py3.9) can be removed after py3.9, see bpo-13601
+    sys.stderr = os.fdopen(sys.stderr.fileno(), "w", 1)
```

Comment:
This works perfectly from what I can tell. I agree with just using `DEBUG` directly...and don't have a strong opinion on where this snippet lands so I'll defer to Matt on that